RecoLocalMuon/Configuration/python/RecoLocalMuonCosmics_cff.py

```diff
@@ -34,11 +34,11 @@
 # CSC sequence
 csclocalreco = cms.Sequence(csc2DRecHits*cscSegments)
 # DT, CSC and RPC together
-muonlocalreco_with_2DSegments = cms.Sequence(dtlocalreco_with_2DSegments+csclocalreco+rpcRecHits)
+muonlocalreco_with_2DSegments = cms.Sequence(dtlocalreco_with_2DSegments+csclocalreco+rpcRecHits+rpcNewRecHits)
 # DT, CSC and RPC together (correct sequence for the standard path)
-muonlocalreco = cms.Sequence(dtlocalreco+csclocalreco+rpcRecHits)
+muonlocalreco = cms.Sequence(dtlocalreco+csclocalreco+rpcRecHits+rpcNewRecHits)
 # DT, CSC and RPC together (with t0seg correction for DTs)
-muonlocalrecoT0Seg = cms.Sequence(dtlocalrecoT0Seg+csclocalreco+rpcRecHits)
+muonlocalrecoT0Seg = cms.Sequence(dtlocalrecoT0Seg+csclocalreco+rpcRecHits+rpcNewRecHits)
```

Comment:
given extensive copy-paste, I think a more practical way is to change rpcRecHits to rpclocalreco and add the new hits there.